package.json

```diff
@@ -56,22 +58,38 @@
     "@babel/plugin-transform-typescript": "7.15.8",
     "@babel/preset-env": "7.15.8",
     "@babel/register": "7.15.3",
+    "@cspell/dict-fullstack": "^2.0.4",
+    "@docusaurus/core": "2.0.0-beta.9",
+    "@docusaurus/preset-classic": "2.0.0-beta.9",
+    "@mdx-js/react": "^1.6.22",
+    "@svgr/webpack": "^5.5.0",
     "@types/chai": "4.2.22",
     "@types/mocha": "9.0.0",
     "@types/node": "16.11.1",
     "@typescript-eslint/eslint-plugin": "5.0.0",
     "@typescript-eslint/parser": "5.0.0",
+    "babel-eslint": "^10.1.0",
```

Comment:
I don't think it's required, can you please check

Reply:
can you please check that we actually use all of new deps, or they are required as `peerDependencies` of docusaurus packages.

Reply:
It is used for linting in `docs/` see the updated eslint config.